data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApi.java

```diff
@@ -13,9 +13,11 @@
 
 package tech.pegasys.teku.beaconrestapi;
 
+import tech.pegasys.teku.infrastructure.async.SafeFuture;
+
 public interface BeaconRestApi {
 
-  public void start();
+  public SafeFuture<?> start();
 
-  public void stop();
+  public SafeFuture<?> stop();
```

Comment:
Tiny nit: We don't need the `public` in these method declarations - all interface methods are public.